crypto/bn/asm/rsaz-2k-avx512.pl

```diff
@@ -525,7 +521,7 @@ sub amm52x20_x1_norm {
 my ($out,$red_tbl,$red_tbl_idx,$tbl_idx) = @_6_args_universal_ABI;
 
 my ($t0,$t1,$t2,$t3,$t4) = map("%ymm$_", (0..4));
-my $t4xmm = $t4 =~ s/%y/%x/r;
+my $t0xmm = $t0 =~ s/%y/%x/r;
 my ($tmp0,$tmp1,$tmp2,$tmp3,$tmp4) = map("%ymm$_", (16..20));
 my ($cur_idx,$idx,$ones) = map("%ymm$_", (21..23));
 
```

Comment:
line 537
a short comment to say rax = (tbl_idx + 4) * 32 == tbl_idx * 2 * 20 or something like that would be helpful (it mightn't be obvious to readers)

Reply:
comment no longer relevant with new implementation